alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/AlertPagedModel.java

```diff
@@ -7,48 +7,19 @@
  */
 package com.synopsys.integration.alert.common.rest.model;
 
+import java.io.Serializable;
 import java.util.List;
 
-import com.synopsys.integration.alert.common.rest.api.ReadPageController;
-
 import net.minidev.json.annotate.JsonIgnore;
 
-public class AlertPagedModel<M extends AlertSerializableModel> extends AlertSerializableModel {
-    public static final Integer DEFAULT_PAGE_NUMBER = Integer.valueOf(ReadPageController.DEFAULT_PAGE_NUMBER);
-    public static final Integer DEFAULT_PAGE_SIZE = Integer.valueOf(ReadPageController.DEFAULT_PAGE_SIZE);
-
-    // FIXME we should use terminology based on "offset" and "limit" which are standard REST API paging terms
-    private final int totalPages;
-    private final int currentPage;
-    private final int pageSize;
-    private final List<M> models;
-
-    public static <M extends AlertSerializableModel> AlertPagedModel<M> empty() {
-        return new AlertPagedModel<>(0, 0, 0, List.of());
-    }
-
+public class AlertPagedModel<M extends AlertSerializableModel> extends AlertPagedDetails<M> implements Serializable {
     public AlertPagedModel(int totalPages, int currentPage, int pageSize, List<M> models) {
-        this.totalPages = totalPages;
-        this.currentPage = currentPage;
-        this.pageSize = pageSize;
-        this.models = models;
-    }
-
-    public int getTotalPages() {
-        return totalPages;
-    }
-
-    public int getCurrentPage() {
-        return currentPage;
-    }
-
-    public int getPageSize() {
-        return pageSize;
+        super(totalPages, currentPage, pageSize, models);
     }
 
+    @Override
     @JsonIgnore
     public List<M> getModels() {
-        return models;
+        return super.getModels();
     }
```

Comment:
Overriding this and calling the super method here in order to add the JsonIgnore annotation so that we ignore the getModels() field when serializing this class.